core/lsp/TypecheckEpochManager.cc

```diff
@@ -13,22 +13,16 @@ void TypecheckEpochManager::assertConsistentThread(optional<thread::id> &expecte
     }
 }
 
-void TypecheckEpochManager::startCommitEpoch(u4 fromEpoch, u4 toEpoch) {
+void TypecheckEpochManager::startCommitEpoch(u4 epoch) {
```

Comment:
Before this change, the preprocessor needed to know the full range of edit epochs contained in the currently typechecking slow path (for reasons related to `TimeTravelingGlobalState`). It was pretty hairy, and confused the definition of 'epoch'.

We now only need to know the epoch of the latest slow path, which corresponds to the epoch of the latest edit in the combined file updates.